cosmic-metrics-collector/src/test/resources/test-data.sql

```diff
@@ -17,3 +17,8 @@ INSERT INTO `service_offering`
   (id, cpu, ram_size)
 VALUES
   (1, 2, 4096);
+
```

Comment:
I'd suggest to move this new test data to a new file. It keeps things easier to maintain as the test cases grow.